ksql-cli/src/test/java/io/confluent/ksql/cli/commands/OptionsTest.java

```diff
@@ -49,4 +49,21 @@ public void shouldReturnEmptyOptionWhenUserAndPassNotPresent() throws Exception
     assertFalse(options.getUserNameAndPassword().isPresent());
   }
 
+  @Test
+  public void shouldReturnFalseIfPasswordIsNull() throws Exception {
```

Comment:
nit: rename to `shouldReturnPasswordNotSetIfPasswordIsNull()` or something similar to clarify what's returning false, and similarly for the other new tests.

Reply:
done